locallib.php

```diff
@@ -1585,7 +1722,41 @@ public function __construct($record) {
     }
 
 }
+/**
+ * Kapselt eine Instanz von ratingallocate_group_choices.
+ * (Encapsulating an instance of ratingallocate_group_choices.)
+ *
+ * @property int $id
+ * @property int $choiceid
+ * @property int $groupid
+ */
+class ratingallocate_group_choices {
```

Comment:
@NinaHerrmann - although you have some classes in here already, if we were following Moodle guidelines correctly, locallib wouldn't include classes - just functions - do you want us to follow your existing structure here, or should we put this new one into a namespaced class file in classes folder?